pkg/util/crypt/crypt_dev.go

```diff
@@ -143,6 +170,10 @@ func (crypt *Device) EncryptFilesystem(path string, key []byte) (string, error)
 	sylog.Debugf("Running %s %s", cmd.Path, strings.Join(cmd.Args, " "))
 	out, err := cmd.CombinedOutput()
 	if err != nil {
+		err = checkCryptsetupVersion(cryptsetup)
+		if err == ErrUnsupportedCryptsetupVersion {
```

Comment:
Should't this be:

```suggestion
		if err != nil {
```

Because there are two other possibilities, `binary path not defined` and `failed to run cryptsetup --version: %s`.